napari/layers/tracks/tracks.py

```diff
@@ -574,17 +574,40 @@ def _norm(p):
         # actually set the vertex colors
         self._track_colors = colormap.map(vertex_properties)
 
-    @property
-    def track_connex(self) -> np.ndarray:
-        """vertex connections for drawing track lines"""
-        return self._manager.track_connex
```

Comment:
You don't mention this change in your PR description, is it removed for the same reason as the track color removal?

Reply:
It's a problem Github's `diff`, this method moved to line 607.